pkgs/by-name/pl/plasticity/package.nix

```diff
@@ -0,0 +1,124 @@
+{ alsa-lib
+, at-spi2-atk
+, autoPatchelfHook
+, cairo
+, cups
+, dbus
+, desktop-file-utils
+, expat
+, fetchurl
+, gdk-pixbuf
+, gtk3
+, gvfs
+, hicolor-icon-theme
+, lib
+, libdrm
+, libglvnd
+, libnotify
+, libsForQt5
+, libxkbcommon
+, mesa
+, nspr
+, nss
+, openssl
+, pango
+, rpmextract
+, stdenv
+, systemd
+, trash-cli
+, vulkan-loader
+, wrapGAppsHook
+, xdg-utils
+, xorg
+}:
+stdenv.mkDerivation rec  {
+  pname = "plasticity";
+  version = "1.4.15";
+
+  src = fetchurl {
+    url = "https://github.com/nkallen/plasticity/releases/download/v${version}/Plasticity-${version}-1.x86_64.rpm";
+    hash = "sha256-wiUpDsfGVkhyjoXVpxaw3fqpo1aAfi0AkkvlkAZxTYI=";
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook autoPatchelfHook rpmextract mesa ];
+
+  buildInputs = [
+    alsa-lib
+    at-spi2-atk
+    cairo
+    cups
+    dbus
+    desktop-file-utils
+    expat
+    gdk-pixbuf
+    gtk3
+    gvfs
+    hicolor-icon-theme
+    libdrm
+    libglvnd
+    libnotify
+    libsForQt5.kde-cli-tools
+    libxkbcommon
+    mesa
+    nspr
+    nss
+    openssl
+    pango
+    stdenv.cc.cc.lib
+    trash-cli
+    vulkan-loader #may help with nvidia users
+    xdg-utils
+    xorg.libX11
+    xorg.libxcb
+    xorg.libXcomposite
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXrandr
+    xorg.libXtst
+  ];
+
+  runtimeDependencies = buildInputs ++ [systemd];
+
+  dontUnpack = true;
+
+  #can't find anything on the internet about these files, no clue what they do
+  autoPatchelfIgnoreMissingDeps = [
+  "ACCAMERA.tx"
+  "AcMPolygonObj15.tx"
+  "ATEXT.tx"
+  "ISM.tx"
+  "RText.tx"
+  "SCENEOE.tx"
+  "TD_DbEntities.tx"
+  "TD_DbIO.tx"
+  "WipeOut.tx"
+   ];
```

Comment:
```suggestion
  # can't find anything on the internet about these files, no clue what they do
  autoPatchelfIgnoreMissingDeps = [
    "ACCAMERA.tx"
    "AcMPolygonObj15.tx"
    "ATEXT.tx"
    "ISM.tx"
    "RText.tx"
    "SCENEOE.tx"
    "TD_DbEntities.tx"
    "TD_DbIO.tx"
    "WipeOut.tx"
   ];
```

Reply:
Can't reply to the other comment - they may be related to c3d headers that make the entire program to be unfree. Here is the creator commenting on a [hacker news thread.](https://news.ycombinator.com/item?id=30695360)